packages/datagateway-table/src/dls/tables/dlsDatasetsTable.component.test.tsx

```diff
@@ -292,6 +293,31 @@ describe('DLS Dataset table component', () => {
     expect(testStore.getActions()[1]).toEqual(fetchDatasetDetailsRequest());
   });
 
+  it('sends off an FetchDatasetSize action when Calculate button is clicked', () => {
+    const { SIZE, ...rowDataWithoutSize } = state.dgcommon.data[0];
+    let newState = state;
+    newState.dgcommon.data[0] = rowDataWithoutSize;
+    const testStore = mockStore(newState);
+
+    let wrapper = mount(
+      <Provider store={testStore}>
+        <MemoryRouter>
+          <DLSDatasetsTable investigationId="1" proposalName="Proposal 1" />
+        </MemoryRouter>
+      </Provider>
+    );
+
+    wrapper
+      .find('[aria-label="Show details"]')
+      .first()
+      .simulate('click');
+
+    const button = wrapper.find('#calculate-size-btn').first();
```

Comment:
@tswinb this can be made into one line if you want `wrapper.find('#calculate-size-btn').first().simulate('click);`.